includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -4023,6 +4024,39 @@ public function get_validated_option( string $field_key, $empty_value = null ) {
+	 *
+	 * @since 10.6.0
+	 * @return array
+	 */
+	public function get_tokens() {
+		$tokens = parent::get_tokens();
+		if ( ! is_user_logged_in() ) {
+			return $tokens;
+		}
+
+		if ( $this->oc_enabled ) {
+			foreach ( $this->get_upe_enabled_payment_method_ids() as $stripe_id ) {
```

PHPMD (2.15.0)

[error] The variable $stripe_id is not named in camelCase. (undefined)

(CamelCaseVariableName)